### PR TITLE
Fix debug mode in fake_quantize_op

### PR DESCRIPTION
**Pull request by qingqing01**
Failed in debug mode since the reduce dimension in Eigen.  Now implemented by max_element instead of Eigen.

### CODE REVIEW

**Review by luotao1 — DISMISSED** [review]
LGTM

File: paddle/fluid/operators/fake_quantize_op.cc [hunk 1]
```diff
@@ -21,26 +21,20 @@ limitations under the License. */
...
+ public:
+  bool operator()(const T a, const T b) { return (std::abs(a) < std::abs(b)); }
+};
 
-template <typename T, int MajorType = Eigen::RowMajor,
-          typename IndexType = Eigen::DenseIndex>
-using ConstEigenVectorArrayMap =
-    Eigen::TensorMap<const Eigen::Tensor<T, 1, MajorType, IndexType>>;
+template struct Compare<float>;
+template struct Compare<double>;
```
Comment (chengduoZH) [inline paddle/fluid/operators/fake_quantize_op.cc]: Line 30~31 is unnecessary.

Comment (qingqing01) [reply paddle/fluid/operators/fake_quantize_op.cc]: Done. Thanks!

**Review by chengduoZH — APPROVED** [review]
LGTM

**Pull request merged**